packages/manager/apps/dedicated/client/app/dedicated/server/servers.controller.js

```diff
@@ -22,65 +29,102 @@ export default class ServersCtrl {
       this.serverStateEnum,
       'server_configuration_state_',
     );
-    this.datacenterEnumFilter = this.getDcEnumFilter(
-      this.datacenterEnum,
-      'server_datacenter_',
+
+    this.datacenterEnumFilter = this.getEnumFilterFromCustomerData(
+      this.dedicatedServers.data,
+      'datacenter',
     );
 
-    this.columnsConfig = [
-      { name: 'name', sortable: this.getSorting('name') },
-      { name: 'reverse', sortable: this.getSorting('reverse') },
-      { name: 'commercialRange', sortable: this.getSorting('commercialRange') },
-      { name: 'datacenter', sortable: this.getSorting('datacenter') },
-      { name: 'state', sortable: this.getSorting('state') },
-    ];
+    this.modelEnumFilter = this.getEnumFilterFromCustomerData(
+      this.dedicatedServers.data,
+      'commercialRange',
+    );
   }
 
   static toUpperSnakeCase(str) {
     return snakeCase(str).toUpperCase();
   }
 
-  getDcEnumFilter(list, translationPrefix) {
-    return {
-      values: reduce(
-        list,
-        (result, item) => {
-          const splittedDcEnumItem = item.split(/(\d+)/);
+  static getMonitoringStatus(monitored, noIntervention) {
+    let monitoring = 'disabled';
+    // proactive intervention
+    if (monitored && !noIntervention) {
+      monitoring = 'proactive';
+    }
+    // no proactive intervention
+    if (monitored && noIntervention) {
+      monitoring = 'no-proactive';
```

Comment:
Maybe you can move the const in constante file for 'disabed', 'proactive', 'no-proactive'